internal/query/org.go

```diff
@@ -77,6 +78,7 @@ type Org struct {
 	ResourceOwner string
 	State         domain_pkg.OrgState
 	Sequence      uint64
+	InstanceID    string
```

Comment:
can we keep the field private?

Reply:
did so, but in the read model i had to make it public as it's in another package